### DISCUSSION TITLE
Guest checkout disabled - proceed to checkout authentication popup is not working

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#145] [1. Help] [answered]
Hello,

I upgraded swissup/breeze to 2.6.0 and swissup/breeze-blank to 2.4.0.
Guest checkout is disabled via Backend → STORES → Configuration → SALES → .Checkout → Allow Guest Checkout  → “No”

Expectation: Upon clicking on "Proceed to Checkout", display authentication popup.

Actual output now: Instead of the popup the cart page refreshes and shows "Guest checkout is disabled".

Steps to replicate:
1. Add a product to cart
2. Go to cart page
3. Click on "Proceed to Checkout" button

Please check. Thank you.

Prior to Breeze version upgrade this feature is working.
[Screencast 2023-05-29 04-57-56.webm](https://github.com/breezefront/community/assets/1543867/cf786387-3a0c-42c5-b1d3-a485f950350a)

**Comment by vovayatsyuk** [comment]
Thank you for reporting! I completely forgot about this popup, and already working on it.

As a workaround, you can disable Breeze on the cart page using the "Disable Breeze for specified URLs" option. Add `/checkout/cart/` line to the field.

**Comment by vovayatsyuk** [comment, marked as answer]
Hi! Please update breeze module and theme. Auth popup is now added to the breeze! https://breezefront.com/updates#june-5-2023
